db/migrate/20220425132605_add_scc_katello_repository_relation.rb

```diff
@@ -0,0 +1,13 @@
+class AddSccKatelloRepositoryRelation < ActiveRecord::Migration[6.0]
+  def change
+    create_table(:scc_katello_repositories) do |t|
+      t.integer :scc_repository_id, null: false
+      t.integer :katello_root_repository_id, null: false
```

Comment:
Is it possible to use `t.references`, here? See https://api.rubyonrails.org/v6.0/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-references